applications/nrf_desktop/doc/config_channel.rst

```diff
@@ -61,37 +98,256 @@ Setting a configuration value of a device, forwarded by the dongle to a paired d
+
+Event ID
+--------
+
+This is the identifier of both module and option for set or fetch operation.
+
+The field is composed of the following subfields:
+
+* Module ID - The application module that should handle the transaction, stored on the 4 most significant bits of the Event ID field.
+* Option ID - The module option the transaction refers to, stored on the 4 least significant bits of the Event ID field.
+
+The values of Module ID and Option ID are assigned during the application build time.
+These values are obtained during the :ref:`device discovery <nrf_desktop_config_channel_device_discovery>` performed by the host.
+The same application module can have different ID value, when it was compiled at different configuration.
+
+Status
+------
+
+The usage of the Status field is different for request and response operations.
+
+Request
+~~~~~~~
+
+In case of request, the value of Status field denotes requested operation.
+The request operations can be grouped as follows:
+
+Generic operations
+  There are two generic operations:
+
+  * ``CONFIG_STATUS_SET``
+  * ``CONFIG_STATUS_FETCH``
+
+  These operations perform, respectively, set or fetch actions related to a given application module and option (as denoted by Event ID).
+  The request is handled by the selected application module.
+  The content and length of the associated data depends on module and option.
+
+Module discovery operations
+  The :ref:`nrf_desktop_info` handles module discovery operations.
+  These operations are used during the :ref:`device discovery <nrf_desktop_config_channel_device_discovery>` to obtain information about the device.
+
+  The Event ID field is not used and it should be set to ``0``.
+  The following operations belong to this group:
+
+  * ``CONFIG_STATUS_GET_MAX_MOD_ID`` - Obtain the maximum value of Module ID supported by the device.
+    The maximum Module ID is returned as a single unsigned byte.
+  * ``CONFIG_STATUS_GET_HWID`` - Obtain an unique Hardware ID of the device.
+    The Hardware ID is represented as 8 bytes.
+  * ``CONFIG_STATUS_GET_BOARD_NAME`` - Obtain the device's board name.
+    The board name is part of the Zephyr board name (:kconfig:`CONFIG_BOARD`) from a beginning to the first underscore (``_``) character.
+    For example the ``nrf52840gmouse_nrf52840`` would return ``nrf52840gmouse`` as the board name.
+
+  .. note::
+    Using :ref:`nrf_desktop_info` is mandatory for every device that is configurable with the configuration channel.
+    The module provides information that are necessary to identify the device.
+
+Recipient discovery operations
+  The :ref:`nrf_desktop_hid_forward` handles recipients discovery operations.
+  These operations are performed to obtain IDs of Bluetooth® LE Peripherals connected to the device.
+  See :ref:`discovering devices connected via dongle <nrf_desktop_config_channel_recipient_discovery>` for more information.
```

Comment:
Latin abbreviations and words are now banned.
Change `via` to `through`.

Reply:
Fixed